src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c

```diff
@@ -3973,36 +3998,36 @@ void KaleidoScope_Update(PlayState* play) {
                     }
                     break;
 
-                case 4:
+                case PAUSE_SAVE_PROMPT_STATE_SAVED:
                     if (CHECK_BTN_ALL(input->press.button, BTN_B) || CHECK_BTN_ALL(input->press.button, BTN_A) ||
                         CHECK_BTN_ALL(input->press.button, BTN_START) || (--D_8082B25C == 0)) {
                         Interface_SetDoAction(play, DO_ACTION_NONE);
                         gSaveContext.buttonStatus[0] = gSaveContext.buttonStatus[1] = gSaveContext.buttonStatus[2] =
                             gSaveContext.buttonStatus[3] = BTN_ENABLED;
                         gSaveContext.hudVisibilityMode = HUD_VISIBILITY_NO_CHANGE;
                         Interface_ChangeHudVisibilityMode(HUD_VISIBILITY_ALL);
-                        pauseCtx->unk_1EC = 5;
+                        pauseCtx->savePromptState = PAUSE_SAVE_PROMPT_STATE_CLOSING_AFTER_SAVED;
                         WREG(2) = -6240;
                         YREG(8) = pauseCtx->unk_204;
                         func_800F64E0(0);
                     }
                     break;
 
-                case 3:
-                case 6:
+                case PAUSE_SAVE_PROMPT_STATE_RETURN_TO_MENU:
+                case PAUSE_SAVE_PROMPT_STATE_RETURN_TO_MENU_ALT:
                     pauseCtx->unk_204 += 314.0f / WREG(6);
```

Comment:
with zero context when I read the enum, "alt" seemed to me like it was going to be an alternative version of the same prompt that did something else. Not just another label for the same thing. Maybe `_2`?